admin_ui/src/components/FormAdd.vue

```diff
@@ -26,24 +44,28 @@ const BASE_URL = process.env.VUE_APP_BASE_URI
 export default Vue.extend({
     props: {
         formSlug: String,
-        schema: Object,
+        schema: Object
     },
     components: {
-        NewForm,
+        NewForm
     },
     data() {
         return {
             errors: "",
-            formData: {},
             formConfig: undefined as FormConfig,
+            successMessage: null
         }
     },
     watch: {
         async formSlug() {
             await this.fetchFormConfig()
-        },
+        }
     },
     methods: {
+        resetForm() {
+            this.$refs.form.reset()
+            this.successMessage = null
+        },
```

Comment:
We need to set `this.errors = ""` because if we first have an error in the form, then we successfully submit the form and then press `Use again` the previous error will be displayed in a new form. Something like this
```vue
resetForm() {
    this.$refs.form.reset()
    this.successMessage = null
    this.errors = ""
},
```

Reply:
Good point - will change it.